src/geo/cartodb-layer-group.js

```diff
@@ -142,6 +142,7 @@ var CartoDBLayerGroup = Backbone.Model.extend({
     if (staticImageURLTemplate) {
       staticImageURLTemplate = this._appendParamsToURL(staticImageURLTemplate, [ 'layer=' + this._getIndexesOfVisibleLayers() ]);
       staticImageURLTemplate = this._appendAuthParamsToURL(staticImageURLTemplate);
+      staticImageURLTemplate = staticImageURLTemplate.replace('{s}', this.getSubdomains()[0]);
```

Comment:
We were trying to avoid the use of a single subdomain, why add this here? Can't getSubdomains be an empty array?